src/main/webapp/app/shared/competency-selection/competency-selection.component.html

```diff
@@ -32,6 +32,23 @@
                                 <fa-icon [icon]="getIcon(competencyLink.competency.taxonomy)" [fixedWidth]="true" />
                                 {{ competencyLink.competency.title }}
                             </label>
+                            @if (checkboxStates[competencyLink.competency.id]) {
+                                <select
+                                    [ngModel]="
+                                        competencyLink.weight < LOW_COMPETENCY_LINK_WEIGHT_CUT_OFF
+                                            ? LOW_COMPETENCY_LINK_WEIGHT
+                                            : competencyLink.weight < MEDIUM_COMPETENCY_LINK_WEIGHT_CUT_OFF
+                                              ? MEDIUM_COMPETENCY_LINK_WEIGHT
+                                              : HIGH_COMPETENCY_LINK_WEIGHT
+                                    "
```

Comment:
These ternary expressions are rather hard to read. If it is not much effort it would be nice if we could use a computed signal for this instead, where the logic might be easier to read.

As the transformation to a computed signal might be rather complex, I do not insist on a change here.